.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# Top-most EditorConfig file
+root = true
+
+# Match and apply these rules for all file
+# types you open in your code editor
+[*]
+# Unix-style newlines
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
```

Comment:
Add a trailing newline at the end of the file here... which this very same config seems to enable.

Reply:
hah yes I didn't have an editorconfig file when I created the file :grinning:

Reply:
Opening and re-saving the file, it automatically inserted it for me though. :+1: